styles.css

```diff
@@ -396,3 +405,57 @@ ul.information-list-subitems {
   color: #fff;
   margin-top: 32px;
 }
+
+.contact-me-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  list-style-type: none;
+  padding: 24px;
+}
+
+.contact-me-form-title {
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-top: 120px;
+  margin-bottom: 73px;
+}
+
+input {
+  border: 0;
+  margin-left: 40px;
+  margin-right: 40px;
+  width: 295px;
+  display: block;
+}
```

Comment:
- [ ]  You need to stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to give inputs some paddings so they can look closely similar to the [original design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471) 😸 .

### Current mobile menu

<div align="center" >
<img src="https://user-images.githubusercontent.com/80895497/148967426-e3798f28-cf01-4a8d-939d-0cad47c1b578.PNG" alt="img" />
</div>